### ISSUE TITLE
Mega-sync failed to sync folder

### ISSUE THREAD

**Issue by erotavlasme** [meganz/MEGAcmd#374]
Hi,
I'm on GNU/Linux 64 bit kernel 5.3.x and I'm trying to use [mega-sync](https://github.com/meganz/MEGAcmd/blob/master/UserGuide.md#sync) command, but I always get `[API:err: time] Failed to sync folder: Already exists.`
```
mega-sync
ID LOCALPATH                REMOTEPATH ActState SyncState SIZE    FILES DIRS
0   path                              path Active       Synced    4.66 GB 332   32  
mega-version
MEGAcmd version: 1.2.0.10: code 1020010
```
Any idea?

**Comment by polmr** [comment]
Are you trying to set a sync to an folder that's being shared to you by any chances? If so, you'd need full permissions to that folder

**Comment by erotavlasme** [comment]
No, the folder is not shared. I found that if I force logout and then login, it works well.
```
mega-logout
mega-login username password
mega-sync localPath remotePath
```

**Comment by didi31** [comment]
Hi,
Same issue on my side.
Did you succeed to make it work?

**Comment by erotavlasme** [comment]
Yes, as described in the [previous message ](https://github.com/meganz/MEGAcmd/issues/374#issuecomment-626172015).
